docs/release-notes/NuGet-6.10.md

```diff
@@ -0,0 +1,178 @@
+
+* Read auditSources from nuget.config files - [#13211](https://github.com/NuGet/Home/issues/13211)
+
+* [Feature]: add command dotnet config set - [#12476](https://github.com/NuGet/Home/issues/12476)
+
+* Warn when vulnerabilities are detected during packages.config restore - [#12307](https://github.com/NuGet/Home/issues/12307)
+
+### Breaking changes
+
+* Add nullable annotations to NuGet.Configuration - [#13250](https://github.com/NuGet/Home/issues/13250)
+
+* [Dotnet Package Search] The search result of the package should be “version” instead of “latestVersion” when executing command “dotnet package search <Package Name> --exact-match --format json” - [#13158](https://github.com/NuGet/Home/issues/13158)
+
+* ResolvedDependencyKey should be struct to reduce memory allocations - [#13138](https://github.com/NuGet/Home/issues/13138)
+
+* [DCR]: Central Package Management - Respect .props file as a way to opt-in to the feature. - [#11834](https://github.com/NuGet/Home/issues/11834)
+
+### Issues fixed in this release
+
+* Stop using JObject in assets file reading to reduce allocations. [#12715](https://github.com/NuGet/Home/issues/12715)
+
+* SourceRepository.GetResourceAsync should be cancellable. - [#13234](https://github.com/NuGet/Home/issues/13234)
+
+* CreateWalkAsync should not be recursive - [#13222](https://github.com/NuGet/Home/issues/13222)
+
+* ProjectModel.HashObjectWriter.OnFlush is using a SHA512 hash versus a cheaper hash which seems like overkill - [#13214](https://github.com/NuGet/Home/issues/13214)
+
+* Owner package metadata is an array in JSON but a string in Protocol types - [#13186](https://github.com/NuGet/Home/issues/13186)
+
+* dotnet package search --verbosity detailed output table too wide - [#13162](https://github.com/NuGet/Home/issues/13162)
+
+* Calls in SetWarningProperties() have allocation overhead due to multiple enumeration - [#13151](https://github.com/NuGet/Home/issues/13151)
+
+* Use of ConcurrentStack in object pool implementation creates large amounts of allocations - [#13147](https://github.com/NuGet/Home/issues/13147)
+
+* IsBestVersion boxes HashSet<T> enumerators resulting in lots of allocation overhead during restore. - [#13146](https://github.com/NuGet/Home/issues/13146)
+
+* Deprecate NuGet.exe list in favor of NuGet.exe search - [#7912](https://github.com/NuGet/Home/issues/7912)
+
+* [Dotnet Package Search] An unhandled exception is thrown when searching with “--verbosity detailed” and “--format json” - [#13300](https://github.com/NuGet/Home/issues/13300)
+
+* `dotnet package search` crashes on .NET 9 preview 2 nightly - [#13286](https://github.com/NuGet/Home/issues/13286)
+
+* Use the StringBuilderPool rather than allocating a new StringBuilder - [#13285](https://github.com/NuGet/Home/issues/13285)
+
+* Pass in an appropriate size for List<T> - [#13284](https://github.com/NuGet/Home/issues/13284)
```

Comment:
It looks like our automated system for generating notes needs to escape HTML chars to `lt;` and `gt;`?  The page still renders, just the `<T>` is missing.

Reply:
Fixed the rendering issues in the entire markdown file.